src/test/java/org/jilt/test/data/annotations/PropagatedAnnotationName.java

```diff
@@ -0,0 +1,23 @@
+package org.jilt.test.data.annotations;
```

Comment:
Same here (please indent this file with 4 spaces, not 2, like the rest of the project).